packages/block-editor/src/components/link-control/style.scss

```diff
@@ -245,6 +245,10 @@ $block-editor-link-control-number-of-actions: 1;
 		border-radius: $radius-block-ui;
 		line-height: 1.1;
 
+		&:focus {
+			box-shadow: none;
+		}
```

Comment:
Can I check this is legit from an a11y standpoint? Focus styles are extremely important to allow folks to understand where focus currently resides. It might be that we're just removing an _additional_ affordance and focus styles still remain in place. If so then we should probably add a comment to explain that, as this code could set a precedent for others in future to remove the remaining focus styles.

Reply:
Looks like the focus ring is still there. I don't see any visual difference.